### DISCUSSION TITLE
Copilot is not working

### DISCUSSION THREAD

**Discussion by plamengj** [community/community#146477] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

My copilot is not working. It said to contact support.

**Comment by SyedAliJunaidSherazi** [comment]
well  here are some troubleshooting options you can try to fix the issue:
1. make sure you are connected to stable connection, sometimes it causes issues.
2. un-install and then re- install your your co pilot extension from vs code or any other ide you are using.
3. restart your ide like vs code or anything.
4. make sure your co-pilot extension is updated with the ide and the versions are compatible.
5. make sure your co pilot is enabled for the account you are using.
6. re verify your subscription or license.
7. signout from your github or copilot account and sign in again.
8. try restarting your ide.

if it didn't worked out, feel free to contact . am here to help. it if solved your problem, please mark my answer as accepted. Do follow me on my github account. :)

**Reply by plamengj** [reply]
Did all that, still have the same problem

**Reply by plamengj** [reply]
I suspect my account is locked. How do I unlock it?

**Comment by davevad93** [comment, marked as answer]
Hi @plamengj ,

It's possible that your account has some kind of restriction with `Copilot`, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select “`Other`” and explain your issue.
